### PR TITLE
[ADD] Refactor + default value

### PR DESCRIPTION
**Pull request by max3903**
* LSP-04-00-1708-13383

### CODE REVIEW

**Comment by max3903** [comment]
@smangukiya 

1. When I create a new attribute and its values, I need to reselect the attribute on the value form view.

attribute value

2. When I create a new attribute and set one of his value as the default, I cannot save the configurable template:

default attribute

I need to save without default and re-edit to set the default value.

**Comment by smangukiya** [comment]
@max3903 : Let me correct it quickly. Thanks!

File: product_configurator_refactor/models/product_attribute.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    value_idss = fields.One2many('product.attribute.value.line',
+                                 'attribute_line_id', 'Values', copy=True)
+    value_ids = fields.Many2many(compute='_get_attribute_values',
+                                 comodel_name='product.attribute.value',
+                                 string='Attribute Values')
+    default_val = fields.Many2one('product.attribute.value',
+                                  compute='_is_default_value',
+                                  string="Default Value", store=True)
+
+    @api.one
```
Comment (max3903) [inline product_configurator_refactor/models/product_attribute.py]: `@api.one` is deprecated: https://github.com/odoo/odoo/pull/8527

Comment (smangukiya) [reply product_configurator_refactor/models/product_attribute.py]: @max3903 : I have pushed couple of commits to resolve all your above mention points.

Updated branch is: 10.0-smangukiya_refactor_product_configurator

Please find it from there. Thanks!

**Comment by max3903** [comment]
@smangukiya 

2 is fixed :+1: 
1 is not. The field is still empty.

**Comment by max3903** [comment]
@smangukiya 

1 fixed as well. Sorry I forgot to reload my browser...

**Pull request merged**